RxCocoa/iOS/Proxies/RxCollectionViewDataSourcePrefetchingProxy.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  RxCollectionViewDataSourcePrefetchingProxy.swift
+//  RxCocoa
+//
+//  Created by Rowan Livingstone on 2/15/18.
+//  Copyright © 2018 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+import UIKit
+import RxSwift
+
+@available(iOS 10.0, tvOS 10.0, *)
+extension UICollectionView: HasPrefetchDataSource {
+    public typealias PrefetchDataSource = UICollectionViewDataSourcePrefetching
+}
+
+@available(iOS 10.0, tvOS 10.0, *)
+let collectionViewPrefetchDataSourceNotSet = CollectionViewPrefetchDataSourceNotSet()
+
+@available(iOS 10.0, tvOS 10.0, *)
+final class CollectionViewPrefetchDataSourceNotSet : NSObject, UICollectionViewDataSourcePrefetching {
+
+    func collectionView(_ collectionView: UICollectionView, prefetchItemsAt indexPaths: [IndexPath]) {}
+
+}
+
+@available(iOS 10.0, tvOS 10.0, *)
+open class RxCollectionViewDataSourcePrefetchingProxy: DelegateProxy<UICollectionView, UICollectionViewDataSourcePrefetching>, DelegateProxyType, UICollectionViewDataSourcePrefetching {
```

Comment:
I think we use a bit different indent. Please take a look at `RxCollectionViewDataSourceProxy`.